### PR TITLE
fix(con-948): optimize hero banner padding & add data-count in cart button

### PR DESCRIPTION
**Pull request by cyrusxzw**
-- Ticket link 
https://aesoponline.atlassian.net/browse/CON-948

--Task
Optimize hero banner padding
Add data-count in cart button

--Solution
use CSS to adjust padding and merge.

### CODE REVIEW

File: src/components/HeroBanner/HeroBanner.module.css [hunk 1]
```diff
@@ -24,12 +24,14 @@
...
+    @media (--viewport-lg) {
+      margin-top: -75px;
+    }
   }
-
-  @media (--viewport-lg) {
-    margin-top: -75px;
-  }
-}
+}
```
Comment (hassanraza0054) [inline src/components/HeroBanner/HeroBanner.module.css:37]: Could you add a new line here? It seems to be a norm in our codebase, so add that back if you can! I've missed this often myself 😅

File: src/components/GlobalNavigation/GlobalNavigation.types.ts [hunk 2]
```diff
@@ -127,6 +127,7 @@ type Clickable = {
 type Trigger = Clickable & {
   onClick: () => void;
   type: 'trigger';
+  count?:number;
```
Comment (hassanraza0054) [inline src/components/GlobalNavigation/GlobalNavigation.types.ts:130]: Lets add a space after the ':' to make it more uniform.

File: src/components/FullWidthHeroScroll/FullWidthHeroScroll.module.css [hunk 3]
```diff
@@ -58,16 +58,18 @@
   }
 }
 
-.newNavLoaded .innerWrapper {
-  @media (--viewport-md) {
-    padding-top: 85px;
-    & h1 {
-      margin-top: -3px;
+:global(.new-navigation) {
```
Comment (hassanraza0054) [inline src/components/FullWidthHeroScroll/FullWidthHeroScroll.module.css:61]: Very nice!

**Review by hassanraza0054 — APPROVED** [review]

File: src/components/GlobalNavigation/GlobalNavigation.types.ts (hunk 2 above)
Comment (MkMan) [reply src/components/GlobalNavigation/GlobalNavigation.types.ts:130]: This should have been done automatically on commit 🤨 🤔

File: src/components/Button/Button.types.tsx [hunk 4]
```diff
@@ -7,6 +7,7 @@ type ButtonProps = {
   aria?: Pick<Aria, 'expanded' | 'haspopup' | 'hidden' | 'label'>;
   children?: ReactNode;
   className?: string;
+  dataCount?: number;
```
Comment (MkMan) [inline src/components/Button/Button.types.tsx:10]: What's this for?

File: src/components/GlobalNavigation/GlobalNavigation.types.ts (hunk 2 above)
Comment (cyrusxzw) [reply src/components/GlobalNavigation/GlobalNavigation.types.ts:130]: It should be auto formatted once I save. Anyway, let me fix it.

File: src/components/Button/Button.tsx [hunk 5]
```diff
@@ -45,6 +46,7 @@ const Button = forwardRef<HTMLButtonElement, ButtonProps>(function ButtonRef(
       aria-hidden={aria?.hidden ?? null}
       aria-label={aria?.label ?? null}
       className={classSet}
+      data-count={dataCount}
```
Comment (MkMan) [inline src/components/Button/Button.tsx:49]: Any non de-structured props are placed on the `button` element anyway. Is this needed? It seems related to a specific use case and not something generic to buttons.

**Review by MkMan — CHANGES_REQUESTED** [review]

File: src/components/Button/Button.types.tsx (hunk 4 above)
Comment (cyrusxzw) [reply src/components/Button/Button.types.tsx:10]: For Adding attribute data-count in Cart. I add dataCount in Button, so it become data-count in html.

**Review by MkMan — APPROVED** [review]

**Pull request merged**